Makefile

```diff
@@ -85,6 +85,7 @@ clean: clean-screenshot-cache clean-offline-docs
 .PHONY: setup
 setup:
 	poetry install
+	poetry install --extras syntax
```

Comment:
You only need `poetry install --extras syntax`, which does both the normal dependencies plus the extras.

Reply:
Will it leave the normal dependencies intact of the extras fail?

Reply:
That I don't know

Reply:
Quickly testing, fresh 3.7 environment, doing the single-line version, stuff installs but you seem to end up with a non-working venv. On the other hand the two-step approach does leave you with a working venv, which is what I was aiming for (you'll still get errors, but at least the venv will be usable).